src/qt/peertablemodel.cpp

```diff
@@ -65,12 +65,19 @@ QVariant PeerTableModel::data(const QModelIndex& index, int role) const
         return QVariant();
 
     CNodeCombinedStats *rec = static_cast<CNodeCombinedStats*>(index.internalPointer());
-
+    const auto time_now{GetTime<std::chrono::seconds>()};
     const auto column = static_cast<ColumnIndex>(index.column());
     if (role == Qt::DisplayRole) {
         switch (column) {
         case NetNodeId:
             return (qint64)rec->nodeStats.nodeid;
+        case Age:
+            if(std::chrono::duration_cast<std::chrono::minutes>(time_now-rec->nodeStats.m_connected) < std::chrono::seconds{60}){
+                /* display connection "Age" in seconds for first minute */
+                return GUIUtil::formatDurationStr(std::chrono::duration_cast<std::chrono::seconds>(time_now-rec->nodeStats.m_connected));
+            }else{
+                return GUIUtil::formatDurationStr(std::chrono::duration_cast<std::chrono::minutes>(time_now-rec->nodeStats.m_connected));
+            }
```

Comment:
d01c0b36 suggestions

- clang-format
- use chrono literals, e.g. `60s` instead of `std::chrono::seconds{60}`
- drop unneeded casts and comment that echoes the code (and shouldn't be in doxygen format here)

<details><summary>suggested diff</summary><p>

```diff 
+using namespace std::chrono_literals;
+
 PeerTableModel::PeerTableModel(interfaces::Node& node, QObject* parent) :
     QAbstractTableModel(parent),
     m_node(node),
@@ -72,11 +75,11 @@ QVariant PeerTableModel::data(const QModelIndex& index, int role) const
         case NetNodeId:
             return (qint64)rec->nodeStats.nodeid;
         case Age:
-            if(std::chrono::duration_cast<std::chrono::minutes>(time_now-rec->nodeStats.m_connected) < std::chrono::seconds{60}){
-                /* display connection "Age" in seconds for first minute */
-                return GUIUtil::formatDurationStr(std::chrono::duration_cast<std::chrono::seconds>(time_now-rec->nodeStats.m_connected));
-            }else{
-                return GUIUtil::formatDurationStr(std::chrono::duration_cast<std::chrono::minutes>(time_now-rec->nodeStats.m_connected));
+            if (time_now - rec->nodeStats.m_connected < 60s) {
+                return GUIUtil::formatDurationStr(time_now - rec->nodeStats.m_connected);
+            } else {
+                return GUIUtil::formatDurationStr(
+                    std::chrono::duration_cast<std::chrono::minutes>(time_now - rec->nodeStats.m_connected));
             }
```
</p></details>

If this becomes larger, say, to display in DD:HH format after 24 hours it may make sense to extract it to a helper function.

Reply:
The ResizeToContents settings should accommodate the duration string as the connection persists longer than 24 hours.

I applied your [suggested diff](https://github.com/bitcoin-core/gui/commit/cca835e812b875b2ed50ecda9c879f0f79b89f00.diff) with attribution.